CHANGELOG.md

```diff
@@ -12,12 +12,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Enabled passing optional arguments to `tox -e docs` calls
 - Logo and banner images
 - Project metadata to pyproject.toml
+- Favicon for homepage
 
 ### Changed
 - Reworked README for GitHub landing page
 - Now has concise contribution guidelines
 - Use Furo theme for documentation
-
+- Fonts in images were converted to paths
```

Comment:
This is what I meant in my other PR: there is no point of adding changelog entries like this one because it does not fix anything that was released before. So just dropping this would make much more sense, since the original "logo and banner" entry would then simply refer to the fixed versions to start with!

Reply:
in some cases this means you cant complete the pipeline due to chagnelog

Reply:
Shouldn't we then adjust the corresponding original entry in the CHANGELOG? Then, all pipelines can stay and we still have a way of seeing in the changelog what happened. Here e.g. one could change the "Logo and banner" entry to "Path-based logo and banner"

Reply:
this si so minor i dont really care at all, just tell me how this one should be fixed and ill derive my future behavior from that

Reply:
done, in this case the other change will also make the pipeline work